core/src/main/java/org/commonjava/indy/core/content/DefaultDownloadManager.java

```diff
@@ -97,6 +97,9 @@
     @Inject
     private StoreDataManager storeManager;
 
+    @Inject
+    private ContentAdvisor contentAdvisor;
```

Comment:
We need to make this Instance<ContentAdvisor>, or possibly Iterable<ContentAdvisor> (I'm not sure if you can @Inject into an Iterable field). Anyway, this will allow us to accommodate multiple advisors.